gui/wxpython/gui_core/pyedit.py

```diff
@@ -508,9 +506,8 @@ def OpenRecentFile(self, path, file_exists, file_history):
         """
         if not file_exists:
             GError(
-                _(
-                    "File <{}> doesn't exist."
-                    "It was probably moved or deleted.".format(path)
+                _("File <{}> doesn't exist. It was probably moved or deleted.").format(
+                    path
```

Comment:
The original string was missing a space between the two sentences. This is one source string change that I think we should keep.

Reply:
Please keep the change.